tests/llvm/casper/arithmetic/ConditionalCount.ll

```diff
@@ -0,0 +1,145 @@
+; ModuleID = 'casper/arithmetic/ConditionalCount.ll'
```

Comment:
Can we remove + gitignore the .ll files? Since we rebuild everything in CI it shouldn't make a difference and helps ensure that we don't have accidental desyncs.

Reply:
done